.travis.yml

```diff
@@ -9,4 +9,6 @@ matrix:
     dist: xenial
     sudo: true
     python: "3.7"
+  - name: "Python: 3.10"
+    python: "3.10-dev"
```

Comment:
this isnt what I want though. People have asked for it but I'm not seeing any movement one here. https://travis-ci.community/t/add-python-3-10/12220 Waiting for it to actually get published before we merge this PR.